faq/tidb-faq.md

```diff
@@ -38,17 +38,7 @@ Yes, it is. When all the required services are started, you can use TiDB as easi
 
 #### How is TiDB compatible with MySQL?
 
-Currently, TiDB supports the majority of MySQL 5.7 syntax, but does not support trigger, stored procedures, user-defined functions, and foreign keys. For more details, see [Compatibility with MySQL](/mysql-compatibility.md).
-
-If you use the MySQL 8.0 client and it fails to connect to TiDB, try to add the `default-auth` and `default-character-set` options:
-
-{{< copyable "shell-regular" >}}
-
-```shell
-mysql -h 127.0.0.1 -u root -P 4000 --default-auth=mysql_native_password --default-character-set=utf8
```

Comment:
I think `--default-character-set=utf8` should not be removed.

Reply:
No, that's fixed too :-)  See https://github.com/pingcap/tidb/issues/17875#issuecomment-660869763 and https://github.com/pingcap/tidb/issues/17875#issuecomment-673019720

Reply:
Ok.